node/integration_test.go

```diff
@@ -8,14 +8,14 @@ import (
 	"testing"
 	"time"
 
-	abci "github.com/tendermint/tendermint/abci/types"
-	"github.com/tendermint/tendermint/libs/log"
-	"github.com/tendermint/tendermint/proxy"
-	"github.com/tendermint/tendermint/types"
 	"github.com/libp2p/go-libp2p-core/crypto"
 	"github.com/libp2p/go-libp2p-core/peer"
 	"github.com/stretchr/testify/mock"
 	"github.com/stretchr/testify/require"
+	abci "github.com/tendermint/tendermint/abci/types"
```

Comment:
How did this happen? Does `gofmt` not run in CI? If not, it should be added (in a separate PR) IMO.

Reply:
No, there is no gofmt in CI. It happened because I switched from vim (with `goimports` on save) to Goland, where my config is yet not fully polished ;)

Reply:
Well, after running `goimports` it seems, this is perfectly fine - `tendermint` is lexicographically after `stretchr`, and `abci` is just an alias.

Reply:
Can we add `goimports`/`gofmt` to CI? It'll make it easier for more contributors to join if they don't have conflicting formatting.